content/how-to/rackspace-email/open-xchange-faq/index.md

```diff
@@ -34,10 +34,21 @@ No, the price you pay will remain the same.
 
 ### The following features are changing: 
 - The maximum forward address limit will change to 4 (Currently 15)
-- Folder cleanup: 
-	- We will implement a new global setting for the spam and trash folders.
-	- The settings for inbox, draft, and sent folders will be removed.
-	- You will be notified if your mailbox is affected by this change.
+- Folder cleanup:
```

Comment:
We probably need a different permalink that will be unique. At the moment we have this article and another article both using `rackspace-email-faq` for permalinks. Might just need to use `open-xchange-faq/`

Reply:
Updated